src/cmpErrorTestExtensions.ts

```diff
@@ -0,0 +1,84 @@
+import prettyFormat from 'pretty-format';
+
+import {CmpError, cmpError, collectCmpErrors, collectCmpErrors4, collectCmpErrors6, isCmpError} from './errors';
+
+const addCmpExtensions = () => {
```

Comment:
These make life *sooooo* much easier for all these repeated checks of whether a value is an error or not.